benchmarks/llama3.1_v6e-256_benchmarks.py

```diff
@@ -0,0 +1,44 @@
+from maxstar_model_configs import llama2_70b_4096
+from maxtext_xpk_runner import BenchmarkRunner
+from maxtext_xpk_runner import HWConfig
+from maxtext_xpk_runner import SWconfig
+from maxtext_xpk_runner import xpk_benchmark_runner
+from maxtext_xpk_runner import XpkConfig
+
+
+DATE = '20241009'
+BASE_DOCKER_IMAGE = 'maxtext_base_image'
+
+ZONE = 'europe-west4'
```

Comment:
Do you mind creating an argparser for the common arguments here: ZONE, PROJECT, CLUSTER_NAME, DEVICE_TYPE, NUM_SLICES so the user can plumb those in when running the script?

Reply:
https://github.com/AI-Hypercomputer/xpk/blob/main/src/xpk/parser/common.py#L26 for reference